tests/SchemaTest.php

```diff
@@ -66,14 +66,12 @@ public function testDrop(): void
 
     public function testBluePrint(): void
     {
-        $instance = $this;
-
-        Schema::table('newcollection', function ($collection) use ($instance) {
-            $instance->assertInstanceOf(Blueprint::class, $collection);
+        Schema::table('newcollection', static function ($collection) {
+            self::assertInstanceOf(Blueprint::class, $collection);
         });
 
-        Schema::table('newcollection', function ($collection) use ($instance) {
-            $instance->assertInstanceOf(Blueprint::class, $collection);
+        Schema::table('newcollection', static function ($collection) {
+            self::assertInstanceOf(Blueprint::class, $collection);
```

Comment:
Alternatively, you can consider importing the PHPUnit functions as we do in various PHPLIB tests. That would allow removing `self::` here.

Reply:
I'm fine with the `self::` prefix, that's more common than global functions.